specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apigateway.json

```diff
@@ -0,0 +1,614 @@
+        }
+      ],
+      "description": "Properties of an API Management gateway resource description."
+    },
+    "ApiManagementGatewayBaseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The current provisioning state of the API Management gateway which can be one of the following: Created/Activating/Succeeded/Updating/Failed/Stopped/Terminating/TerminationFailed/Deleted.",
+          "readOnly": true
+        },
+        "targetProvisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the API Management gateway, which is targeted by the long running operation started on the gateway.",
+          "readOnly": true
+        },
+        "createdAtUtc": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation UTC date of the API Management gateway.The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.",
+          "readOnly": true
+        },
+        "frontend": {
+          "$ref": "#/definitions/FrontendConfiguration",
+          "description": "Information regarding how the gateway should be exposed."
+        },
+        "backend": {
+          "$ref": "#/definitions/BackendConfiguration",
+          "description": "Information regarding how the gateway should integrate with backend systems."
+        },
+        "configurationApi": {
+          "$ref": "#/definitions/ConfigurationApi",
+          "description": "Information regarding the Configuration API of the API Management gateway. This is only applicable for API gateway with Standard SKU."
+        }
+      },
+      "description": "Base Properties of an API Management gateway resource description."
+    },
+    "ApiManagementGatewayListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiManagementGatewayResource"
+          },
+          "description": "Result of the List API Management gateway operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of API Management services."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The response of the List API Management gateway operation."
+    },
+    "ApiVersionConstraint": {
+      "type": "object",
+      "properties": {
+        "minApiVersion": {
+          "type": "string",
+          "description": "Limit control plane API calls to API Management gateway with version equal to or newer than this value.",
+          "example": "2019-01-01"
+        }
+      },
+      "description": "Control Plane Apis version constraint for the API Management gateway."
+    },
```

Comment:
dont need this.

Reply:
This was removed yesterday, thanks for confirming